README.md

```diff
@@ -71,7 +75,7 @@ Esri welcomes contributions from anyone and everyone. Please see our [guidelines
 
 ## Licensing
 
-Copyright 2013 Esri
+Copyright 2020 Esri
```

Comment:
I'm not sure if this line should be updated, but I saw .NET and [Android](https://github.com/Esri/arcgis-runtime-samples-android/commit/c6e07a8fc7165029b7dd57745f9eac2f6b97d682) updates their year info.